cli/tests/pcluster/configure/test_pcluster_configure.py

```diff
@@ -237,6 +253,46 @@ def test_no_automation_no_awsbatch_no_errors(mocker, capsys, test_datadir):
     _verify_test(mocker, capsys, output, error, config, TEMP_PATH_FOR_CONFIG)
 
 
+def test_no_input_no_automation_no_errors_with_config_file(mocker, capsys, test_datadir):
```

Comment:
this seems very similar to the second one, I think we can have a single parametrized test